shopfloor_reception/services/reception.py

```diff
@@ -1237,15 +1262,24 @@ def _post_shopfloor_created_line(self, selected_line):
         )
 
     def _auto_post_line(self, selected_line):
-        new_move = selected_line.move_id.split_other_move_lines(
-            selected_line, intersection=True
-        )
-        if new_move:
-            # A new move is created in case of partial quantity
-            new_move.extract_and_action_done()
-            return
-        # In case of full quantity, post the initial move
-        selected_line.move_id.extract_and_action_done()
+        # If user only processed 1/5 and is the only one working on the move,
+        # then selected_line is the only one related to this move.
+        # In such case, we must ensure there's another move line with the remaining
+        # quantity to do, so selected_line is extracted in a new move as expected.
+        if selected_line.product_uom_qty:
+            selected_line.product_uom_qty = 0
```

Comment:
Safer to do that on any line having a product_uom_qty. Especially that before your change in this PR, the lines had a computed product_uom_qty.
```suggestion
        lines_with_qty_todo = selected_line.move_id.move_line_ids.filtered("product_uom_qty")
        if lines_with_qty_todo:
          lines_with_qty_todo.product_uom_qty = 0
```

If you have a line with 1/5 and another user creates a line with 1/0 then he cannot post his line unless you change the other initial line

Reply:
done